src/vivarium_uw_covid/components/seiir_beta.py

```diff
@@ -37,7 +37,79 @@ def beta_predict(coeffs, covs):
     return pd.DataFrame(np.exp(log_beta), index=covs.index)
 
 
-def make_alternative_covariates(df_covs, loc_id, **alt_cov_values):
+# from https://github.com/ihmeuw/covid-model-seiir-pipeline/blob/master/src/covid_model_seiir_pipeline/core/utils.py#L128
+def beta_shift(beta_fit: pd.DataFrame,
```

Comment:
I just updated the master branch.  You'll want the newest version of this algorithm from there.

Settings are still being modified as Bobby does some predictive validity work.  The most recent production run settings had a window size of 28  and the residual averaging distributed as U[1, 21].  We're testing now with residual averaging at U[14, 56] which should be more stable from run to run.

Reply:
I think I'll wait until there is an updated set of estimates and then I'll update the code here, too.  I want to match what we have on the covid19.ihme webpage.